### PR TITLE
Jest/server

### PR DESCRIPTION
**Pull request by BrentTB**
Added more tests for sever.js

### CODE REVIEW

File: src/server.js
Comment (RudolphTorlage) [inline src/server.js:1]: This has removed a lot of code, have you exported the functionality to functions that simply get called by the sockets events?

File: tests-jest/userController.test.js
Comment (RudolphTorlage) [inline tests-jest/userController.test.js:1]: removed unnecessary logs

File: tests-jest/server.test.js
Comment (RudolphTorlage) [inline tests-jest/server.test.js:1]: Added jest test coverage for server file.

**Review by RudolphTorlage — COMMENTED** [review]
Please confirm changes to server.js file

File: src/server.js
Comment (BrentTB) [reply src/server.js:1]: yes, and this allows us to test the socket functionality

**Review by RudolphTorlage — APPROVED** [review]
Functionality confirmed.

**Pull request merged**